### PR TITLE
feat(appeals/web): BOAT-838 Update valid date input page to show date incorrect date values in the form fields when validation flags an error 

### PR DESCRIPTION
**Pull request by glpins**
## Describe your changes

[BOAT-838](https://pins-ds.atlassian.net/browse/BOAT-838)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.controller.js [hunk 1]
```diff
@@ -86,11 +86,20 @@ const renderValidDatePage = async (request, response, apiErrors) => {
 	}
 	const { appealId, appealReference } = request.session;
 
-	const mappedPageContent = updateValidDatePage(appealId, appealReference);
-
+	let dateValidDay = request.body['valid-date-day'];
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.controller.js:89]: These should be `const` as there's no later modification of the values - in fact, these variables aren't strictly necessary (you could just pass `request.body['valid-date-day']` etc. directly into the mapper function)

**Review by james-hill-kc — APPROVED** [review]
Looks good, just one minor comment from me, too minor to block approval

**Review by james-hill-kc — APPROVED** [review]
LGTM!

File: appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.controller.js (hunk 1 above)
Comment (glpins) [reply appeals/web/src/server/appeals/appeal-details/appellant-case/outcome-valid/outcome-valid.controller.js]: 👍

**Pull request merged**